core/src/test/java/pt/webdetails/cpf/utils/AbstractCorsUtilTest.java

```diff
@@ -0,0 +1,106 @@
+/*!
```

Comment:
Can you run `mvn clean install site` and add coverage for the untested lines? The report will be generated in **cpf/core/target/site/jacoco/pt.webdetails.cpf.utils/AbstractCorsUtil.html**.